wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -69,3 +70,20 @@ def visit_any_function(self, node: AnyFunctionDef) -> None:
         self._check_decorators(node)
         self._check_magic_methods(node)
         self.generic_visit(node)
+
+
+class WrongParentClassListVisitor(BaseNodeVisitor):
+    """Check parent class list."""
+
+    def visit_ClassDef(self, node: ast.ClassDef) -> None:
+        """Check 'object' class in parent list."""
```

Comment:
You are missing `Raises:` policy here. We list all errors that are thrown by this visitor in its docstring.